src/codec/builtin/FixSizedTypesCodec.ts

```diff
@@ -18,6 +18,36 @@ import * as Long from 'long';
 import {BitsUtil} from '../../util/BitsUtil';
 import {UUID} from '../../core/UUID';
 
+// Taken from long.js, https://github.com/dcodeIO/long.js/blob/master/src/long.js
+
+/**
+ * @type {number}
```

Comment:
This comment looks redundant as it provides no additional information. Information for types is already available since we use TypeScript. The same applies to the following constants.

Reply:
yes